views/events/calls_events.view.lkml

```diff
@@ -205,22 +210,118 @@ view: calls_events {
     sql: ${TABLE}."UUID_TS" ;;
   }
 
+  dimension_group: active_user_date {
+    label: "Active User"
+    description: "Use with Active User/Instance Dimensions to enable Daily, Weekly & Monthly active user/instance functionality with this explore."
+    type: time
+    timeframes: [date, week, month, year, fiscal_quarter, fiscal_year, day_of_week, fiscal_month_num,
+      week_of_year, day_of_year, day_of_week_index, month_name, day_of_month, fiscal_quarter_of_year]
+    sql: ${dates.date_date}::date ;;
+
+  }
+
   measure: instance_count {
     label: "Instance Count"
     type: count_distinct
     sql:  ${user_id};;
     drill_fields: []
   }
 
+  measure: user_count {
+    label: " Total Users"
+    type: count_distinct
+    sql:  ${participant_id};;
+    drill_fields: []
+  }
+
+  measure: last_call_started_date {
+    label: " Last Call Started Date"
```

Comment:
Do we intentionally leave a space in front here and elsewhere in this file?

Reply:
Yeah, it is the auto-complete feature, caused when you hit `tab` when it suggests code, it doesn't make a difference.